### PR TITLE
add create action in order controller

### PR DESCRIPTION
**Pull request by shahad-yousif**

### CODE REVIEW

File: app/controllers/api/v1/orders_controller.rb [hunk 1]
```diff
@@ -19,4 +19,34 @@ def show
     render json: {order: order}.merge(general_response: general_response), action_name: 'show', status: :ok
   end
 
+  def create
+    customer = Customer.find(params[:customer_id])
+    customer_info = customer.customer_info
+    cart = customer.cart
+    cart_total_price = Helper.display_token_value(cart.total_price)
```
Comment (ssharba) [inline app/controllers/api/v1/orders_controller.rb]: we changed Helper class name

Comment (shahad-yousif) [reply app/controllers/api/v1/orders_controller.rb]: @samer in pure_api still not change

Comment (MwMaki) [reply app/controllers/api/v1/orders_controller.rb]: @shahad-yousif It did

Comment (ssharba) [reply app/controllers/api/v1/orders_controller.rb]: yes, check dev branch

**Comment by shahad-yousif** [comment]
i fixed it and and Pundit error solved too

File: app/controllers/api/v1/orders_controller.rb [hunk 2]
```diff
@@ -19,4 +19,34 @@ def show
...
+    cart_total_price = HelperMethods.display_token_value(cart.total_price)
+    TokenTransaction.create(tran_process: 'Withdrawal', tran_unit: 'Token', customer_id: customer.id, amount: cart_total_price, tran_type: 'buy_product')
+    customer_info.update!(token_balance: customer_info.token_balance - cart_total_price)
+    total_products = cart.order_items
+    total_products.each do |product|
+      product_after_purchase = Product.find_by(id: product.product_id)
+      product_after_purchase.update_attributes(quantity: product_after_purchase.quantity - product.quantity)
+    end
+    if cart.order_items.count != 0 && cart.total_price < customer_info.token_balance
+      order = Order.new(customer_id: customer.id, order_number: create_order_number)
```
[the remark this answers is not in the document]
Comment (shahad-yousif) [reply app/controllers/api/v1/orders_controller.rb]: we don't use it in tracking but i think we still need it

**Comment by samer** [comment]
You got the wrong Samer. Please stop mentioning me.

File: app/controllers/api/v1/orders_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (shahad-yousif) [reply app/controllers/api/v1/orders_controller.rb]: @ssharba we used order number in order index page the search depend on this column also when order still not process by admin will not have group number have only order number

**Comment by ssharba** [comment]
@shahad-yousif can you share response son here and URL

**Pull request merged**